docker-compose-alpine.yaml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+
+services:
+  graphdb:
+    container_name: graphdb
+    image: khaller/graphdb-free:latest
+    network_mode: host
+
+
+  mysql:
+    container_name: mysql
+    image: mysql:8.0.17
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: 1
+      MYSQL_ROOT_PASSWORD: null
+      MYSQL_DATABASE: operationaldb
+    expose:
+      - 3306
+    network_mode: host
+
+  ot-node:
+    container_name: ot-node
+    image: origintrail/ot-node:v6_release_testnet_alpine
+    depends_on:
+      - graphdb
```

Comment:
We can add here also mysql, because it depends on it also

Reply:
mysql also added to depends